core/include/irods/private/http_api/openid.hpp

```diff
@@ -14,12 +14,35 @@
 
 namespace irods::http::openid
 {
+	enum class TokenType
```

Comment:
`TokenType` -> `token_type`

Please update references in documentation too.

Reply:
Done.